### PR TITLE
Fix doc generation and validation failures

### PR DESCRIPTION
**Pull request by rachelcarmena**
PR on `release/0.11.0` branch to fix failures when generating and validating the documentation from #130.

### CODE REVIEW

File: arrow-fx/src/main/kotlin/arrow/fx/Promise.kt [hunk 1]
```diff
@@ -103,7 +103,7 @@ interface Promise<F, A> {
    *       p.complete(2)
    *     }
    *   }.attempt().unsafeRunSync() ==
-   *     IO.raiseError<Int>(Promise.AlreadyFulfilled).attempt().unsafeRunSync()
+   *     IO.raiseException<Int>(Promise.AlreadyFulfilled).attempt().unsafeRunSync()
```
Comment (pakoito) [inline arrow-fx/src/main/kotlin/arrow/fx/Promise.kt]: This...doesn't seem right. The generic should be of the type of `Promise.AlreadyFulfilled` instead, right?

Comment (pakoito) [reply arrow-fx/src/main/kotlin/arrow/fx/Promise.kt]: Oh nevermind, it's an attempt. Ye.

**Comment by nomisRev** [comment]
@rachelcarmena I can continue this. I was also looking at it during the weekend:D

**Comment by rachelcarmena** [comment]
> @rachelcarmena I can continue this. I was also looking at it during the weekend:D

Thanks @nomisRev !!! :raised_hands: Just 2 remaining failures :clap: Thanks for your help!!

File: settings.gradle [hunk 2]
```diff
@@ -1,7 +1,6 @@
 rootProject.name = 'arrow-fx-lib'
 
 include 'arrow-fx'
-include 'arrow-docs'
```
Comment (nomisRev) [inline settings.gradle]: Whoops

Comment (rachelcarmena) [reply settings.gradle]: Don't worry @nomisRev :heart:

**Pull request merged**